src/skins/vector/css/common.css

```diff
@@ -264,6 +250,12 @@ input[type=text]:focus, textarea:focus {
     color: white;
 }
 
+.emojione {
```

Comment:
I haven't changed this, the Matrix SDK is still rendering them as `emojione`, but I agree